### PR TITLE
Added resources for Japanese language

### PR DESCRIPTION
**Pull request by mikoto-studio**

### CODE REVIEW

File: src/content/community/language-resources/index.md [hunk 1]
```diff
@@ -109,3 +109,9 @@ If you are bilingual and want to help us reach more people, you can also get inv
 - [Tino Group](https://wiki.tino.org/ethereum-la-gi/) - overview of Ethereum, dApps, wallets and FAQs
 - [Tap Chi Bitcoin](https://tapchibitcoin.io/tap-chi/tin-tuc-ethereum-eth) - web platform with subpages for Ethereum news and education
 - [Coin68](https://coin68.com/ethereum-tieu-diem/) - cryptocurrency portal with Ethereum news and educational content
+
+### Japanese {#ja}
```
Comment (minimalsm) [inline src/content/community/language-resources/index.md:113]: We need to add a space here for formatting

```suggestion
### Japanese {#ja}

```

File: src/content/community/language-resources/index.md [hunk 2]
```diff
@@ -109,3 +109,9 @@ If you are bilingual and want to help us reach more people, you can also get inv
...
+- [Japan Virtual and Crypto assets Exchange Association](https://jvcea.or.jp/)
+- [Japan Cryptoasset Business Association](https://cryptocurrency-association.org/)
+- [Crypto Investment Guide for Beginners](https://www.crypto-currencies.jp/)
```
Comment (minimalsm) [inline src/content/community/language-resources/index.md:116]: I don't think we want to include general crypto investment advice websites here. 
```suggestion
```

File: src/content/community/language-resources/index.md [hunk 3]
```diff
@@ -109,3 +109,9 @@ If you are bilingual and want to help us reach more people, you can also get inv
...
+- [Wikipedia - Ethreum](https://ja.wikipedia.org/wiki/%E3%82%A4%E3%83%BC%E3%82%B5%E3%83%AA%E3%82%A2%E3%83%A0)
```
Comment (minimalsm) [inline src/content/community/language-resources/index.md:117]: This link is broken but not sure linking out to Wikipedia is all that useful?

```suggestion
```

Comment (wackerow) [reply src/content/community/language-resources/index.md:117]: Agree here... in my opinion this is a link people already know how to get to, and could likewise be placed on every single language. Given that, I don't think it adds value listing this here.

**Review by wackerow — CHANGES_REQUESTED** [review]
@mikoto-studio Hey, thanks for submitting this! Much appreciated. I do agree with the points @minimalsm made about being hesitant on linking out to investment advice pages. The links we provide should be educational in nature surrounding Ethereum itself, and personally would argue against introducing stuff outside of the scope of this site. Open for debate on this though.
The other being the wikipedia link. Wikipedia is extremely well known and this could equally be listed under every language if we wanted to go that route, which I don't think is particularly valuable. 

Going to make these corrections for now and pull this in. Feel free to comment though if you disagree with the adjustments and we can always discuss more and make another PR if needed 👍🏻

**Review by wackerow — APPROVED** [review]

**Comment by wackerow** [comment]
@all-contributors please add @mikoto-studio for content

**Comment by minimalsm** [comment]
@mikoto-studio be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by mikoto-studio** [comment]
Thank you for inviting to discord and your reviews! It's good learning opportunity for me.
Since there's still not so many good resources in Japanese, I think this is really good project;)

**Comment by kurotaky** [comment]
👍

**Pull request merged**